config/jest/setupTests.ts

```diff
@@ -21,3 +21,11 @@ if (process.env.CI) {
     consoleLog(message);
   };
 }
+
+// Mock focus-trap-react to prevent errors in tests that use modals. focus-trap-react is a package used
+// by LeafyGreen and is not a direct dependency of Spruce.
+jest.mock("focus-trap-react", () => {
```

Comment:
im so mad that we have to do this 😒 but nice job!

also nit
```suggestion
// by LeafyGreen and is not a direct dependency of Parsley.
jest.mock("focus-trap-react", () => {
```